tests/helics/application_api/FederateTests.cpp

```diff
@@ -103,15 +103,15 @@ TEST(federate_tests, bad_broker_error_zmq_ci_skip)
     helics::FederateInfo fi(helics::core_type::ZMQ);
     fi.coreInitString = "--broker=b1 --tick=200 --timeout=800 --networktimeout=400";
 
-    EXPECT_THROW(std::make_shared<helics::Federate>("test1", fi), helics::RegistrationFailure);
+    EXPECT_THROW(auto fed = std::make_shared<helics::Federate>("test1", fi), helics::RegistrationFailure);
```

Comment:
What was the warning the compiler gave for this?

Reply:
Visual studio 2019 has a [[nodiscard]] specifier on std::make_shared so this was producing a warning since the return value wasn't being captured even though in this case it would throw.